docs/format/status.rst

```diff
@@ -0,0 +1,373 @@
+| create            | ✓     | ✓     | ✓     | ✓          |
++-------------------+-------+-------+-------+------------+
+| add vertex        | ✓     | ✓     | ✓     | ✓          |
++-------------------+-------+-------+-------+------------+
+| remove vertex     |       |       |       |            |
++-------------------+-------+-------+-------+------------+
+| add edge          | ✓     | ✓     | ✓     | ✓          |
++-------------------+-------+-------+-------+------------+
+| remove edge       |       |       |       |            |
++-------------------+-------+-------+-------+------------+
+| get name          | ✓     | ✓     | ✓     | ✓          |
++-------------------+-------+-------+-------+------------+
+| get vertex        | ✓     | ✓     | ✓     | ✓          |
++-------------------+-------+-------+-------+------------+
+| get edge          | ✓     | ✓     | ✓     | ✓          |
++-------------------+-------+-------+-------+------------+
+| add extra info    |       |       |       |            |
++-------------------+-------+-------+-------+------------+
+| remove extra info |       |       |       |            |
++-------------------+-------+-------+-------+------------+
+| get extra info    | ✓     |       |       |            |
++-------------------+-------+-------+-------+------------+
+| check validation  | ✓     |       |       |            |
++-------------------+-------+-------+-------+------------+
+| serialize         | ✓     | ✓     | ✓     | ✓          |
++-------------------+-------+-------+-------+------------+
+| deserialize       | ✓     | ✓     | ✓     | ✓          |
++-------------------+-------+-------+-------+------------+
+
+
+Notes:
+
+* \(1) Through fastFFI bindings to the GraphAr C++ library.
+
+* \(2) Through py4j bindings to the GraphAr spark library.
+
+
+Libraries Version Compatibility
+-------------------------------
++-------------------+-------+-------+-------------------+
+| GraphAr C++       | C++   | CMake | Format Version    |
+| Version           |       |       |                   |
++===================+=======+=======+===================+
+| 0.11.x            | 17+   | 2.8+  | 1.0.0             |
++-------------------+-------+-------+-------------------+
+
++----------------+----------+--------+------------------+
+| GraphAr Java   | Java     | Maven  | Format Version   |
+| Version        |          |        |                  |
++================+==========+========+==================+
+| 0.1.0          | 1.8, 11  | 3.6+   | 1.0.0            |
++----------------+----------+--------+------------------+
+
++----------------+--------------+---------+---------+----------------+
+| GraphAr Spark  | Apache Spark | Scala   | Java    | Format Version |
+| Version        | Version      | Version | Version |                |  
++================+==============+=========+=========+================+
+| 0.1.0          | 3.2.x-3.3.x  | 2.12.x  | 1.8, 11 | 1.0.0          |
```

Comment:
Java version should be 1.8, and 11 is not supported now.

Reply:
ok

Reply:
not available, see #277

Reply:
sorry, it's only for java library, it's 1.8

Reply:
OK， I got it.